http/serve.py

```diff
@@ -0,0 +1,143 @@
+# ASGI main
```

Comment:
Can we add a `READM` , and perhaps also state on the relation to the _Asynchronous Server Gateway Interface_ ?

Reply:
Sure.  This is the first of a few PRs.  One of the latter will have readme, etc. if that's OK